simd/src/Kokkos_SIMD.hpp

```diff
@@ -253,6 +285,13 @@ using host_abi_set    = abi_set<simd_abi::scalar, simd_abi::avx2_fixed_size<4>,
                              simd_abi::avx2_fixed_size<8>>;
 using data_type_set =
     data_types<std::int32_t, std::int64_t, std::uint64_t, double, float>;
+#elif defined(KOKKOS_ARCH_ARM_SVE)
+using host_abi_set =
+    abi_set<simd_abi::scalar, simd_abi::sve_fixed_size<2>,
+            simd_abi::sve_fixed_size<4>, simd_abi::sve_fixed_size<8>,
+            simd_abi::sve_fixed_size<16>>;
```

Comment:
```suggestion
using host_abi_set =
    abi_set<simd_abi::scalar, simd_abi::sve_fixed_size<__ARM_FEATURE_SVE_BITS / 32>,
            simd_abi::sve_fixed_size<__ARM_FEATURE_SVE_BITS / 64>>;
```
If I'm not mistaken, depending on the SVE vector length there will always be two of those abis which would lead to compile errors if used, e.g. on Grace, `basic_simd<T, sve_fixed_size<8>>` and `basic_simd<T, sve_fixed_size<16>>` would be undefined.